core/src/main/java/nl/nn/adapterframework/util/CompactSaxHandler.java

```diff
@@ -116,6 +116,7 @@ private void printCharData(boolean startElement) throws SAXException {
 		moveElementFound = moveElementFound
 				&& context != null
 				&& !startElement
+				&& (chompLength < 0 || length <= chompLength)
```

Comment:
I agree, it would be a bit silly to chomp and move it... 😮

Reply:
Please blame the persons who invented this. I just restored the way it worked before and spend some time figuring that out. 
I think it makes sense to define a max character size and use that for all character data. No exceptions. That's how it worked and how I now made it again. It is in line with the descriptions of Receiver e.g. 
So now, we don't chomp and move it @nielsm5. It just checks the length and if it is too long, it is not moved, but cut off. I think it actually is consistent and makes sense.

Reply:
Secondly, I think you both read the logic wrong. 
`&& (chompLength < 0 || length <= chompLength)`
So that means that it is fine if chompLength is not set OR character data size is <= chompLenght (if set).

Reply:
I'm not blaming you Jakobjob :-) I understand you are restoring previous behaviour, I just happen to think the previous behaviour does not make sense. :-)

Reply:
Ah yes 😉 But if I read the above statements "then we effectively never move" and from Niels "it would be a bit silly to chomp and move it", I thought otherwise, because both are not true. It is just not moved if it is too long, which can be a valid safety measure. But yes, we're just thinking here about what the intentions of the functionality should be. I'll continue below.